lms/djangoapps/teams/static/teams/js/spec/views/teams_tab_spec.js

```diff
@@ -0,0 +1,92 @@
+define([
```

Comment:
It was just moved, I think. I don't know how the history got lost again... I'll try to recover it.